binary_transparency/firmware/api/witness.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
super nit: the dates in the new files should be 2021.

Reply:
All New files have now 2021 in the HEADER